packages/dcos-integration-test/extra/test_dcos_diagnostics.py

```diff
@@ -348,16 +348,13 @@ def test_dcos_diagnostics_bundle_create_download_delete(dcos_api_session):
 def _check_diagnostics_bundle_status(dcos_api_session):
     # validate diagnostics job status response
     diagnostics_bundle_status = check_json(dcos_api_session.health.get('/report/diagnostics/status/all'))
-    required_status_fields = ['is_running', 'status', 'errors', 'last_bundle_dir', 'job_started', 'job_ended',
-                              'job_duration', 'diagnostics_bundle_dir', 'diagnostics_job_timeout_min',
+    required_status_fields = ['is_running', 'status', 'last_bundle_dir', 'job_started',
+                              'diagnostics_bundle_dir', 'diagnostics_job_timeout_min',
                               'journald_logs_since_hours', 'diagnostics_job_get_since_url_timeout_min',
                               'command_exec_timeout_sec', 'diagnostics_partition_disk_usage_percent',
                               'job_progress_percentage']
 
     for _, properties in diagnostics_bundle_status.items():
-        assert len(properties) == len(required_status_fields), 'response must have the following fields: {}'.format(
```

Comment:
Again, just curious why this is being removed?

Reply:
https://github.com/dcos/dcos/pull/5704#discussion_r292880694